src/transformStateWithClones.js

```diff
@@ -7,7 +7,33 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        currentState = {};
```

Comment:
don't reassign new object, please use `delete` properties instead

Reply:
@Kacper7001 still waiting for the fix, we want to delete every field from `currentState`